admin/acceptance.py

```diff
@@ -950,6 +950,39 @@ def eliot_output(message):
     sys.stdout.write(format % message)
     sys.stdout.flush()
 
+def capture_upstart(reactor, host, output_file):
+    """
+    SSH into given machine and capture relevant logs, writing them to
+    output file.
+
+    :param reactor: The reactor.
+    :param bytes host: Machine to SSH into.
+    :param file output_file: File to write to.
+    """
+    # TODO: check the format we need to use here (if we need it at all)
```

Comment:
Printing this line isn't useful; you're writing logs from multiple nodes to same file, so it'll just be interleaved and add no useful info.